plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/bean/CoinbaseBeanV1.java

```diff
@@ -54,6 +58,8 @@ public void setTransactionType(String value) {
             transactionType = BUY;
         } else if ("Send".equalsIgnoreCase(value)) {
             transactionType = WITHDRAWAL;
+        } else if (List.of("Receive", "Rewards Income").contains(value)) {
```

Comment:
its better to define these static lists as static final at class level. Then it ts not created everytime.